eng/common/scripts/Package-Properties.ps1

```diff
@@ -162,6 +162,13 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
     $allPackageProperties = Get-AllPkgProperties
     $diff = Get-Content $InputDiffJson | ConvertFrom-Json
     $targetedFiles = $diff.ChangedFiles
+    # The exclude paths and the targeted files paths aren't full OS paths, they're
+    # GitHub paths meaning they're relative to the repo root and slashes are forward
+    # slashes "/". The ExcludePaths need to have a trailing slash added in order
+    # correctly test for string matches without overmatching. For example, if a pr
+    # had files sdk/foo/file1 and sdk/foobar/file2 with the exclude of anything in
+    # sdk/foo, it should only exclude things under sdk/foo.
+    $excludePaths = $diff.ExcludePaths | ForEach-Object { $_ + "/" }
```

Comment:
What happens if they already end in a slash? It might be worth updating to `$_.TrimEnd("/") + "/"`

Reply:
Will push another commit with these changes.